### PR TITLE
Allow configuration of checkout_branch branch command

### PR DESCRIPTION
**Pull request by tapayne88**
## Check list

- [x] I have performed a self-review of my code
- [ ] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

## Description

Love the project! I'm using it to replace some scripts I'd hand rolled doing roughly the same thing!

However, I was hoping to use `checkout_branch` to quickly swap between recent branches. To do this I'd like to pass `--sort=-committerdate` to the `git branch` command to sort them by most recent commit.

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Refactor
- [ ] Breaking change
- [ ] Documentation change

## Test environment

- Shell
    - [ ] bash
    - [x] zsh
    - [ ] fish
- OS
    - [x] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Others:

### CODE REVIEW

File: bin/git-forgit
Comment (tapayne88) [inline bin/git-forgit:450]: Move `--all` to the default value so it can be omitted if desired

**Review by cjappl — APPROVED** [review]
Seems super useful. I'll probably end up using that option as well! 

Great idea. Will wait for a review from one of the other folks before merging

**Comment by carlfriedrich** [comment]
Thanks for your contribution @tapayne88. That sounds super useful indeed! I won't have time for a detailed check within the next two weeks, though, @cjappl.

**Comment by cjappl** [comment]
Great addition. Thanks for the contribution

**Pull request merged**
